src/main/kotlin/Network.kt

```diff
@@ -24,5 +24,50 @@ class Network(val capacity: Int, val listOfCars: MutableList<Car>) {
             car.isDelayed = overCapacity
         }
     }
-
+    fun CarsDevidedByCapacity():Double{
+        val x = (listOfCars.size / capacity).toDouble()
+        return x
+    }
+fun switchCase(x:Double):Double {
+    var y :Double = 0.0
+    if (x > 1.5) {
+        y = 0.9
+    }
+    if ((1.25 <= x) && (x <= 1.5)) {
+        y = 0.7
+    }
+    if ((1.1 <= x) && (x <= 1.25)) {
+        y = 0.5
+    }
+    if ((0.9 <= x) && (x <= 1.1)) {
+        y = 0.3
+    }
+    if ((0.7 <= x) && (x <= 0.9)) {
+        y = 0.2
+    }
+    if ((0.5 <= x) && (x <= 0.7)) {
+        y = 0.1
+    }
+    if (x < 0.5) {
+        y = 0.05
+    }
+    return y
```

Comment:
Can you please use self-explaining names for variables? It gets confusing with x,y,z